### PR TITLE
Check brute force limit in nearest neighbor blueprint.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

**Review by geirst — APPROVED** [review]
👍

**Comment by geirst** [comment]
@arnej27959 FYI

File: searchlib/src/vespa/searchlib/queryeval/nearest_neighbor_blueprint.h [hunk 1]
```diff
@@ -23,6 +23,7 @@ class NearestNeighborBlueprint : public ComplexLeafBlueprint {
     std::unique_ptr<vespalib::tensor::DenseTensorView> _query_tensor;
     uint32_t _target_num_hits;
     bool _approximate;
+    bool _use_brute_force;
```
Comment (arnej27959) [inline searchlib/src/vespa/searchlib/queryeval/nearest_neighbor_blueprint.h:26]: i don't see the need to have _use_brute_force as another flag; just update _approximate with "false" instead.

**Review by arnej27959 — COMMENTED** [review]
please consider dropping the extra flag.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**